zagar_server/src/main/java/messageSystem/MessageSystem.java

```diff
@@ -1,22 +1,25 @@
 package messageSystem;
 
-import main.MasterServer;
 import main.Service;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.*;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
 
 /**
  * @author e.shubin
  */
 public final class MessageSystem {
   private final static Logger log = LogManager.getLogger(MessageSystem.class);
 
-  private final Map<Address, ConcurrentLinkedQueue<Message>> messages = new HashMap<>();
+  private final Map<Address, BlockingQueue<Message>> messages = new HashMap<>();
```

Comment:
What is the reason to declare  `Map<Address, BlockingQueue<Message>>` instead of `Map<Address, Queue<Message>>` ?